### PR TITLE
Update shell container to python3

### PR DESCRIPTION
**Pull request by scottaubrey**
Updated the shell container used to create a linux environment to run builder on OS X to use Python 3.

Also updated the `update.sh` script to use the `python3` binary name, which looking at other examples in repo, is the expected binary name too.

### CODE REVIEW

**Comment by scottaubrey** [comment]
FYI @NuclearRedeye

**Comment by NuclearRedeye** [comment]
Tested locally and works fine for me. @lsh-0 should take a look to the. `update.sh` script though just to comment on its compatibility before we get this merged.

**Comment by lsh-0** [comment]
> @lsh-0 should take a look to the. update.sh script though

Yup, I don't want to be a bottleneck but please cc me in on all changes to builder. I don't necessarily need to review trivial changes, it just helps me keep my mental model current.

File: update.sh
Comment (lsh-0) [inline update.sh:5]: Nice catch, I overlooked this in the recent py2-removal. 

On Ubuntu at least `python` will default to `python2`. I wouldn't be surprised if osx did the same. I run Arch and `python` points at the most recent version of python installed:

```
$ python --version
Python 3.10.1
```

**Review by lsh-0 — APPROVED** [review]
@scottaubrey , I'm happy to merge any changes to builder that are waiting on my approval and don't need any further feedback.

You can ask me to merge it or I can do it automatically.

I won't merge draft PRs but happy to review them.

**Pull request merged**
